app/controllers/manuals_controller.rb

```diff
@@ -5,33 +5,49 @@ class ManualsController < ApplicationController
 
   before_action :ensure_manual_is_found
   before_action :ensure_document_is_found, only: :show
-  before_action :load_manual
   before_action :prevent_robots_from_indexing_hmrc_manuals
 
   def index
-    set_expiry(manual)
+    set_expiry(content_store_manual)
+
+    render :index, locals: {
+      presented_manual: presented_manual
+    }
   end
 
   def show
-    set_expiry(document)
-    @document = Document.new(document, @manual)
+    set_expiry(document_from_repository)
+    document = Document.new(document_from_repository, manual)
+
+    render :show, locals: {
+      presented_manual: presented_manual,
```

Comment:
I like this, making the dependencies to the view explicit is :+1:.  I wonder though if we could call this param `manual` instead though?  It being `presented_manual` doesn't feel that useful in the context of the view (there's no other manual variable to compare it against, and the view doesn't need to know that this is a presenter based manual instance).  If we call it `manual` we remove a bunch of the changes in the view files for this commit.